lib/ansible/utils/import_module.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2020 Matt Martz <matt@sivel.net>
```

Comment:
If we plan on keeping this long term, let's put this in `module_utils/compat/import_module.py`  If we plan on getting rid of this as soon as we can actually stop hacking the controller to work on python-2.6, let's name it something that shows it shouldn't be used by others and when it should be removed.  Maybe: `utils/_compat26.py` ?

Reply:
Done.